webCommander/centralHub/views.py

```diff
@@ -25,26 +26,36 @@ def submit_x_user(request):
         form_handle = SubmitXUser()
     return render(request, 'tweetFramework/x_handle_submission_form.html',{'form_handle':form_handle})
 
-
 def list_twitter(request):
     alltweets = TwitterUser.objects.all()
 
+
     return render(request, 'tweetFramework/showtweets.html',{'alltweets':alltweets})
 
 def handle_summary(request, twitter_handle):
-    handle = get_object_or_404(TwitterUser, twitter_handle=twitter_handle)
+    user = TwitterUser.objects.get(twitter_handle=twitter_handle)
+
+    #TODO Get posts from database and render them
```

Comment:
need help here? as I am not subscribed to their API, maybe you can provide some data in a django fixture so I can load that to test this function?

https://docs.djangoproject.com/en/5.0/howto/initial-data/

Reply:
this should do:

```
(venv) √ webCommander (X_post_display) $ mkdir webCommander/fixtures && python manage.py dumpdata centralHub > webCommander/fixtures/centralhub-data.json
```

then commit the file, then I can load in the twitter data :)